hvplot/util.py

```diff
@@ -252,13 +260,12 @@ def process_crs(crs):
     errors = []
     if isinstance(crs, str) and crs.lower().startswith('epsg'):
         try:
-            crs = crs[5:].lstrip().rstrip()
-            return ccrs.epsg(crs)
+            crs = pyproj.CRS.from_epsg(crs).to_wkt()
```

Comment:
Again, is this needed? There is no need to do extra calculations if the simple case works.

Reply:
The simple case unfortunately does not work